### PR TITLE
Completed the first phase of documentation

### PR DESCRIPTION
**Pull request by milutinke**
Greetings,

As mentioned in #2 I've completely written a very detailed documentation using the Markdown format and [Vue Press 2.0](https://github.com/vuepress/vuepress-next).
Everything is compiled to `HTML`  so it should be able to load on the documentation page once it is merged.
I've gone very in detail but I've also attached some quick bits for people who do not want details.

In the Phase 2 I plan to make a script for generating the C# API reference for `ChatBot.cs` from the code and to record detailed You Tube Tutorials for each Chat Bot and section.
I've done my best to check for spelling and grammatical mistakes, I hope that none have slipped by.

Here is a video of how the documentation looks like:
https://streamable.com/33xfux 

As for the maintaining the documentation, it's really simple, there are 2 commands and everything is written in Markdown just like on Git Hub.
Also, a really neat thing is that the Localization support is really easy and simple to use, someone just needs to make a folder with the language code and copy over the files and change the content in them.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      type: secondary
+actionText:
+actionLink:
+features:
+    - title: Chat
+      details: Chat with other people on the server
+    - title: Automation
+      details: Create bots to do automated tasks
+    - title: Supported Versions
+      details: 1.8 - 1.19
```
Comment (ORelio) [inline docs/README.md]:
```suggestion
      details: 1.4 - 1.19
```

File: docs/guide/installation.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Installation
+
+-   [You Tube Tutorials](#youtube-tutorials)
```
Comment (ORelio) [inline docs/guide/installation.md]:
```suggestion
-   [YouTube Tutorials](#youtube-tutorials)
```

File: docs/guide/installation.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+-   [Installation on Windows by Daenges](https://www.youtube.com/watch?v=BkCqOCa2uQw)
+-   [Installation on Windows + Auto AFK and More by Dexter113](https://www.youtube.com/watch?v=FxJ0KFIHDrY)
+
+## Download a compiled binary
+
+#### Windows
+
+In order to run **Minecraft Console Client** on Windows you need the following software:
+
+-   [.NET Framework 4.8 or new-er](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
```
Comment (ORelio) [inline docs/guide/installation.md]: Either 4.0 or greater (original version)
Or 6.0 or greater (new version https://github.com/MCCTeam/Minecraft-Console-Client/pull/1674)
(Seems like documentation below states 6.0 so you can just go for 6.0 as the original version will be discontinued)

**Review by ORelio — APPROVED** [review]
Good job, that's a great improvement compared to the existing state of documentation!
Please let me know when the PR is ready to merge.
I could not go in-depth in every paragraph but made a few notes by the way:

File: docs/guide/installation.md (hunk 3 above)
Comment (milutinke) [reply docs/guide/installation.md]: Implemented the requested changes.
I think that I've done everything that I wanted so far, it can be merged.

**Comment by ORelio** [comment]
Thanks for your work! 👍

**Pull request merged**